CHANGELOG.md

```diff
@@ -1,3 +1,34 @@
+# v0.18.0 (next release)
+
+## Breaking changes
+- `sample_with` can no longer be passed to `.sample()`. Instead, the user has to rerun
+`.build_posterior(sample_with=...)`. (#573)
+- the `posterior` no longer has the the method `.sample_conditional()`. Using this 
+  feature now requires using the `sampler interface` (see tutorial
+  [here](https://www.mackelab.org/sbi/tutorial/07_conditional_distributions/)) (#573)
+- Posterior saved under `sbi` `v0.17.2` or older can not be loaded under `sbi` 
```

Comment:
I'd suggest we make this the first point since it's likely the most inconvenient change for users